docker-compose.yml

```diff
@@ -26,9 +26,9 @@ x-extra-hosts:
 
 services:
   db:
-    image: postgres:11.16
+    image: postgres:12.17
```

Comment:
@mitodl/devops Could anyone tell if this could cause any problems in the deployments? I don't think we use the docker like this in our server. 

I'm asking this because locally it did cause problems and I had to dump and restore the database because of incompatble files issues between these versions.

Reply:
This won't cause us any problems in RC or production.